### ISSUE TITLE
footnotes and citations don't play nicely

### ISSUE THREAD

**Issue by trigaten** [trigaten/Learn_Prompting#79]
Footnotes overwrite citations at the bottom of the page.

**Comment by antsanchez** [comment]
This is easily solved by marking the footnotes with a name instead of the note. For instance, on docs/applied_prompting/mc_tutorial.md, instead of marking the first note as "[^1]", mark as "[^note]", as on the following image:
<img width="773" alt="Bildschirmfoto 2023-01-10 um 18 32 41" src="https://user-images.githubusercontent.com/9377770/211621876-0873c6ba-44d9-42f6-a41e-13bd85a27f35.png">

**Comment by trigaten** [comment]
> This is easily solved by marking the footnotes with a name instead of a number. For instance, on `docs/applied_prompting/mc_tutorial.md`, instead of marking the first note as `[^1]`, mark as `[^note]`, as on the following image:
> <img alt="Bildschirmfoto 2023-01-10 um 18 32 41" width="773" src="https://user-images.githubusercontent.com/9377770/211621876-0873c6ba-44d9-42f6-a41e-13bd85a27f35.png">

This still seems undesirable. Additionally, where will that footnote appear WRT the citations. IMO it would be ideal to have all footnotes appear first and then citations at the bottom of the page. What are your thoughts on this?

**Comment by antsanchez** [comment]
We are talking about this plugin, right? https://www.npmjs.com/package/@benchmark-urbanism/remark-bibtex

**Comment by trigaten** [comment]
> We are talking about this plugin, right? https://www.npmjs.com/package/@benchmark-urbanism/remark-bibtex

Yes, it isn't very robust, so I forked it for what this site needed: https://github.com/trigaten/remark-bibtex

**Comment by antsanchez** [comment]
Ok. This plugin is deprecated, should we first try with this other https://github.com/timlrx/rehype-citation?

**Comment by antsanchez** [comment]
@trigaten about my last comment, should I try firs with  https://github.com/timlrx/rehype-citation or directly with https://github.com/trigaten/remark-bibtex?

**Comment by trigaten** [comment]
> @trigaten about my last comment, should I try firs with https://github.com/timlrx/rehype-citation or directly with https://github.com/trigaten/remark-bibtex?

Preferably try https://github.com/trigaten/remark-bibtex since I have already slightly edited it to the site's needs, but whatever you think is best!
